docs/release_notes_toc.md

```diff
@@ -0,0 +1 @@
+# OpenScienceLab Release Notes
```

Comment:
This seems like an extraneous file... No content, and not obviously linked to from anywhere.